guides/v2.1/cloud/docker/docker-development-testing.md

```diff
@@ -0,0 +1,119 @@
+---
+group: cloud-guide
+title: Functional testing in Docker
+functional_areas:
+  - Cloud
+  - Docker
+  - Configuration
+---
+
+You can use the `{{site.data.var.ct}}` package to run functional tests in the Docker environment. Functional tests are in the `src/Test/Functional/Acceptance` folder. See an example in the [ece-tools repository](https://github.com/magento/ece-tools/tree/develop/src/Test/Functional/Acceptance).
+
+## Prerequisites
+
+Before you run functional tests, you must prepare your environment with the following steps.
+
+1.  Stop all services that use the following ports:
+
+    -  `80`—varnish
+    -  `443`—web, tls
+    -  `3306`—apache, mysql
+
+1.  Switch to the preferred PHP version for running tests.
```

Comment:
Would be helpful to provide hint here --Consider providing link to [Versions](https://devdocs.magento.com/guides/v2.3/cloud/docker/docker-config.html#service-versions), or provide example command